backend_plugin/plugin.xml

```diff
@@ -30,34 +30,6 @@
             sequence="M1+6">
       </key>
```

Comment:
Can we remove some of this other stuff?

ui.handlers is no longer used and should probably be removed
ui.bindings is for a hotkey that can be removed (You can use ctrl+6 or cmd+6 to start the plugin)

I'm not sure if we need ui.commands, but we should see if we can remove that

Reply:
That's a good point, I've removed the rest of the unused plugin extensions and it seems fine now